### ISSUE TITLE
Ore Generation Too Heavy

### ISSUE THREAD

**Issue by Rayzilla68** [TeamFRM/TAIGA#102]
I am finding TAIGA ores EVERYWHERE. They seem to spawn more than iron sometimes and at all world levels. They need to be much rarer and at lower levels. It makes base building on a new world very difficult with all the virtually unbreakable ore blocks this generates. 

Yes, I know I can make changes in the .config files, but anyone first using this won't know that and once the chunks are generated, it is too late.

**Comment by Zkaface** [comment]
@Rayzilla68 Use the configuration gui.
Also the config is by default set to a low generation, so that there could not be so many taiga ore instead of iron. 

Which version do you use?

**Comment by Rayzilla68** [comment]
I am using 1.1.4 for 1.10.2. I did not change any configurations when I loaded it up since I assumed I wouldn't have to. So, I am playing at the defaults right now and the generation is HIGH. Literally Vibranium everywhere at y=64. As I get closer to bedrock, it is riddled with Meteorite and Mythril. After just four hours I came away with six blocks of Eternite after smelting. I only got 4 blocks of iron. 

I assume it is too late to change the configuration for this world since the area I am playing has already generated.

**Comment by Rayzilla68** [comment]
Looking at the .config file, all ores are set to "1.0."

**Comment by Zkaface** [comment]
@Rayzilla68 Try the newest build instead. The old config was a big mess :-\ but be warned, that a lot of ores are replaced with other ones. The book pages as a guide for tinkering is coming in the next days. Actually I work on a well arranged layout.

**Comment by Rayzilla68** [comment]
Vanilla ores were replaced? I don't understand. Is that why I find a crap ton of slag gold and no gold? Lignite and not coal?

**Comment by Zkaface** [comment]
@Rayzilla68 No, version 1.2.x does replace a lot of taiga ores with new ones

**Comment by Zkaface** [comment]
@Zkaface And it changes a lot in general

**Comment by Rayzilla68** [comment]
I still don't understand. I asked if TAIGA is replacing vanilla ores with its own and you replied saying that 1.2 changes TAIGA ores with new ores? Whut?

If there is a lot of replacing of vanilla ores, what do I do in the config to keep the vanilla generation the same and make the TAIGA ores more rare? I am also using Terrain Control which jumbles ore generation around, so I can't afford to have a diminished amount of vanilla ore. 

Because you are so responsive and I just started my world this weekend without much hard progress, I will do the newest version and restart the world. At least it is working with the HermitPack now. :)

By the way, I noticed the change to the Materials and You book. Thanks for adding that. TAIGA adds so much it is hard to know what is what.

**Comment by Cheezy16** [comment]
1.2.x for TAIGA is the update that changes all ore compositions and most alloys. Basically what Zkaface is trying to say is if you reload the world that you generated in 1.1.4, once you update your world will have tons of holes from where old ores that were removed once were. Some ores don't get replaced, but he is saying that you should just make a new world for 1.2.4 when you update.

**Comment by Rayzilla68** [comment]
That's just what I am doing now. Hopefully no big changes causing me to start a new world will happen. I took the "play a world for a year" challenge on Reddit. :)

**Comment by Zkaface** [comment]
@Rayzilla68 Sorry, I mixed up my thoughts. 

None of the taiga releases change vanilla ores. But the version 1.2 changed some of the old taiga ores to new ones. Like mythril. I did this to prevent some duplicates with other mods.

**Comment by Rayzilla68** [comment]
OK, gotcha. Thanks!

**Comment by Rayzilla68** [comment]
Ugh...I put in the newest version, started a new world and...vibranium for days, meteorite spawning in veins of 15+ blocks. You should warn people that they'll need to adjust the spawn rates in the config before starting a world. Add to that there is no clear way to get to a pick hard enough to break some of these ores and its a real pain in the butt. 

In my opinion, none of these ores should show up above level 12 in the Overworld. I am finding some of them at the tops of mountains. It makes building a base very frustrating when you try to dig out a lower level and find 60-ish unbreakable blocks in a relatively small area. 

I am either going to have to give myself a vibranium pick head or start the world all over again with no TAIGA. Double ugh.

**Comment by Zkaface** [comment]
@Rayzilla68 At first there is a way to get to the highest tier, but yes, it is hard to reach. On the other hand I don't know how you could get a high spawn rate, when all the variables got another name. 

To be on the safe side you should remove the old config file and restart the game. 

The last clue... Yes. Taiga adds various ores which are hard to mine. I apologize for having problems, finding enough space to mine and build, but this seems to happen if there are broken variables in the config. 

My conclusion: remove taiga. remove the old config. Restart the game and remove old blocks. Add taiga again and use simple retrogen to add the ores in my intended way.

**Comment by IdleGandalf** [comment]
There is no need to remove TAIGA when he already is on the latest version and started a new world. Maybe you should reset your configuration to default (by removing the config file in /config/) and generate a new world. If you still have the impression that there is too much ore spawning you can report back here and we have to look into the default values again.

**Comment by Zkaface** [comment]
What @IdleGandalf said! So closed for now. Please open another issue, if the problem persists.
